src/main/java/TaskTwo.java

```diff
@@ -12,49 +12,48 @@ public static void main(String[] args) {
         char[] chars = toCharArray(str);
         char[] unfoldedChars = unfoldedChars(chars);
         String unfoldStr = new String(unfoldedChars);
-//        String unfoldStr = String.valueOf(unfoldedChars);
         System.out.println(unfoldStr);
     }
 
-    public static char[] toCharArray(String inputString) {
-        return inputString.toCharArray();
+    public static char[] toCharArray(String getUserInput) {
+        return getUserInput.toCharArray();
     }
 
     public static char[] unfoldedChars(char[] chars) {
         char[] unfoldedCharsArray = new char[chars.length];
```

Comment:
one more solution with a simple for loop

'public static char[] unfoldedChars(char[] chars) {
        char[] unfoldedCharsArray = new char[chars.length];
        for (int i = chars.length; i > 0; i--) {
            unfoldedCharsArray[chars.length - i] = chars[i-1];
        }
        return unfoldedCharsArray;
    }'